### ISSUE TITLE
Gepetto background image only

### ISSUE THREAD

**Issue by olivier-stasse** [jmirabel/hpp-gui#4]
Dear Joseph,
Unfortunately I do not have the same window that the one indicated in the video specified by the README.md file. The only thing I see is the Gepetto logo image in full precision...
It might be a plugin not properly compiled or loaded.
Here is my settings.conf file:
[plugins]
libhppcorbaserverplugin.so=true
libremoteimuplugin.so=false
libhppwidgetsplugin.so=true

**Comment by jmirabel** [comment]
The video hasn't  updated yet. It is normal that you do not have the blue screen.
Load a robot or click on File>Create view and it should appear.

The technical details:
The widget that shows the 3D scene

**Comment by jmirabel** [comment]
BTW, to see if the plugins are properly loaded, see `Tools` > `Plugins`

**Comment by olivier-stasse** [comment]
Thanks. Got it.
But unfortunately, when I try to load hrp2-14 from the predefined description, I have a bluscreen without the robot, and the following error:
"No Node named "hrp2_14/"

When trying pr2, I have a dialog box with "File not fond" with the following string:
"/opt/ros/hydro/share"

Thanks again for your reactivity.

**Comment by jmirabel** [comment]
The paths in your `robots.conf` are probably wrong. Here are some hints:
* For PR2:
  * Are you using ROS Hydro ?
  * Does `/opt/ros/hydro/share/pr2_description` exists ?
* For HRP2:
  * Did you replace `${CMAKE_INSTALL_PREFIX}` by the installation path of HRP2 ?
  * In the installation directory, the following path must exists:
    * `/share/hrp2_14_description/urdf/hrp2_14.urdf`
    * `/share/hrp2_14_description/srdf/hrp2_14.srdf`

Are you able to load HRP2 with the `gepetto-viewer` ?

**Comment by olivier-stasse** [comment]
Thanks  I managed to make it work with PR-2
The correct path was:
/opt/ros/indigo/share
And the package path was:
/home/ostasse/devel/ros-indigo-1/install/share/hpp_tutorial

I tried with the package path for hrp2_14:
/home/ostasse/devel/ros-indigo-1/install/share/hrp2_14_description
The robot is loaded, the tree can be seen in the body-tree dialog box, but the robot is not visible.
Sounds like I am missing something obvious...

**Comment by olivier-stasse** [comment]
For the meshes directory I put:
/home/ostasse/devel/ros-indigo-1/install/share/

**Comment by jmirabel** [comment]
* What happen if you right-click on the hrp2 root node and select the first option of the `Attach to window` sub-menu ?

* Are you trying to load both robots in the same session ? I theory, it works but I have never tested it.
* Does the `Log` window tell you something ?
* Do you have any output in the terminal where you launched the executable ? Very likely, you have plenty. Most likely, errors and warnings will contain "not found" or "already exists".

**Comment by jmirabel** [comment]
I pushed some changes on the `devel` branch that should solve some of the bugs you encountered. I guess it won't solve the problems regarding collada files.

You will probably need to set the `BUILD_HPP_MANIPULATION_WIDGETS_PLUGIN` cmake variable to `OFF` in order to compile that version. The `devel` branch goes along with some changes that are not yet in the HPP `devel` branch.

**Comment by jmirabel** [comment]
The problem has been reproduced on @jcarpent computer. It seems the problem is related to Qt and OSG have some conflicts.
Using `osgconv` to convert collada to osg format and then loading the osg file works just fine.

A possible work around we discussed is to do this conversion semi-automatically.

**Comment by olivier-stasse** [comment]
Hi Joseph,
Thanks for your help. I have been using the pure gepetto viewer. I fixed some problems related to the display of multiple visuals and I am now able to display correctly multiple robots (around 8) directly from the urdf file. For this issue, I discussed with @jcarpent. Thanks for the workaround, I will try to understand where does come from the problem if I have some time.
Thanks again for your support.
